src/style.css

```
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 700;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  margin: 20px auto;
+  display: flex;
+  justify-content: center;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+  
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
+
+.label {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.label_big:after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
```

```suggestion
  background-image: url("./images/Search.svg");
  background-repeat: no-repeat;
  background-size: contain;
```
ці три властивості спільні. Їх можна додати до `.label:after`